modules/table/src/testFixtures/java/org/apache/ignite/distributed/ItTxTestCluster.java

```diff
@@ -275,6 +282,10 @@ public class ItTxTestCluster {
 
     protected String localNodeName;
 
+    private Map<String, Map<ZonePartitionId, ZonePartitionRaftListener>> zonePartitionRaftGroupListeners = new HashMap<>();
+
+    private Map<String, Map<ZonePartitionId, ZonePartitionReplicaListener>> zonePartitionReplicaListeners = new HashMap<>();
```

Comment:
```suggestion
    private final Map<String, Map<ZonePartitionId, ZonePartitionRaftListener>> zonePartitionRaftGroupListeners = new HashMap<>();

    private final Map<String, Map<ZonePartitionId, ZonePartitionReplicaListener>> zonePartitionReplicaListeners = new HashMap<>();
```

Reply:
Are these maps always accessed from the same thread?

Reply:
Yes, there are plenty of other maps in the class like
```
    protected Map<String, ReplicaManager> replicaManagers;

    protected Map<String, RaftGroupOptionsConfigurer> raftConfigurers;

    protected Map<String, LogStorageFactory> logStorageFactories;
```
that are common HashMaps
```
        replicaManagers = new HashMap<>(nodes);
        replicaServices = new HashMap<>(nodes);
        txManagers = new HashMap<>(nodes);
```
and are used in a similar manner